### ISSUE TITLE
Combine FISMA system and business application inventories - create one master inventory

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/EDX#194]
In order to eliminate duplication and confusion, the FISMA system/subsystem inventory in Archer is being compared to the business application inventory in GEAR. The goal is to have one inventory, since there is so much overlap between the two.

**Comment by AKM399** [comment]
@hatfieldjm4 can we epic some of these related issues together?

**Comment by hatfieldjm4** [comment]
> @hatfieldjm4 can we epic some of these related issues together?

This has gained a lot of steam in the last month due to an SES performance plan metric wanting to report adoption of cloud services in a more accurate way. I am nearly done with our initial scrub of combining the two inventories-- I just have to verify with our security POC so he can make the changes in his system. 

TLDR: this is almost done and will be uploaded to the dev GEAR site (https://dev1.ea.gsa.gov)

**Comment by AKM399** [comment]
@hatfieldjm4 is this closable??
